tests/silent/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -24,7 +24,7 @@ $value = @     $cache[$key];
 <?php
 $variable = @foo();
 /* Intentional file error */
-$my_file = @file('non_existent_file') or
+($my_file = @file('non_existent_file')) or
```

Comment:
I think it is not bug, it is feature, why:
```php
$c = false or foo();
```

Because `and` and `or` have less precedence, we assign `false` and call `foo()`, without parens, it is very misleading and some developers can confused, with parens all everything is clear